sdk/vision/azure-ai-vision-imageanalysis/mypy.ini

```diff
@@ -7,7 +7,3 @@
 # See GitHub issue: https://github.com/Azure/autorest.python/issues/2317
```

Comment:
I suspect you don't need this mypy.ini file at all anymore

Reply:
Good point @kristapratico! removed.